.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
     ],
     "eslint.quiet": true,
     "files.eol": "\n",
-    "github-pr.targetBranch": "dev"
+    "github-pr.targetBranch": "dev",
+    "editor.codeActionsOnSave": {
```

Comment:
is this something we all want in our editors?

Reply:
I think this was a merge from dev

Reply:
base branch hasn't been updated